dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/SecondOrder.mo

```diff
@@ -0,0 +1,115 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block SecondOrder
+  import Modelica;
+  import Dynawo.Types;
+
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Real D "Damping";
+  parameter Real K = 1 "Gain";
+  parameter Real W "Angular frequency";
```

Comment:
w without a capital letter for frequency. + units

Reply:
w and Angular frequency in 1/s